src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -163,10 +163,11 @@ public function update_item( $request ) {
 			return $fields_update;
 		}
 
-		return $this->prepare_item_for_response(
-			get_post( $request['id'] ),
-			$request
-		);
+		wp_after_insert_post( $post, true, $post_before );
```

Comment:
This will need the change to `wp_update_post()` to change the third parameter to false. It's outside the diff so I can't note it there.

Reply:
Done!